common/components/multi-file-upload/multi-file-upload.js

```diff
@@ -116,7 +116,24 @@ MultiFileUpload.prototype = {
           row.parentNode.removeChild(row)
           this.render()
         })
-        .catch(() => false)
+        .catch(error => {
+          let errorMessage = 'could not be deleted'
+          if (error.response && error.response.data) {
+            errorMessage = error.response.data
+          }
+          const row = target.closest('.dz-success')
+          row.classList.remove('dz-success')
+          row.classList.add('dz-error')
+          row.querySelector('[data-dz-errormessage]').innerHTML = errorMessage
+          const buttons = row
+            .closest('.app-multi-file-upload__list')
```

Comment:
Just spotted this is using presentation classnames still.

Having a quick look at the element and the module I think this selector might be more appropriate:

```javascript
const buttons = this.dropzone.previewsContainer.querySelectorAll('button[data-dz-remove]')
```

I also noticed there is a `js-` class in that isn't actually being used so we could tidy that element up and just remove that class (`js-upload-delete`) from the template:

![image](https://user-images.githubusercontent.com/3327997/77931043-8ed85780-72a3-11ea-98e6-064c96290399.png)

Reply:
done